### PR TITLE
Cleanup descriptions and help information

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes https://github.com/projectatomic/libpod/issues/1166

File: cmd/podman/pod_rm.go [hunk 1]
```diff
@@ -22,12 +22,13 @@ var (
...
 	}
-	podRmDescription = "Remove one or more pods"
-	podRmCommand     = cli.Command{
-		Name: "rm",
-		Usage: fmt.Sprintf(`podman rm will remove one or more pods from the host. The pod name or ID can be used.
-                            A pod with containers will not be removed without --force.
-                            If --force is specified, all containers will be stopped, then removed.`),
+	podRmDescription = fmt.Sprintf(`
+podman rm will remove one or more pods from the host. The pod name or ID can
+be used.  A pod with containers will not be removed without --force.            If --force is specified, all containers will be stopped, then removed.
```
Comment (TomSweeneyRedHat) [inline cmd/podman/pod_rm.go]: does the tab/spaces before the 'If' look OK?

Comment (rhatdan) [reply cmd/podman/pod_rm.go]: Nope, supposed to be a new line.

**Comment by TomSweeneyRedHat** [comment]
LGTM, assuming happy tests.

**Comment by mheon** [comment]
LGTM once tests are green

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**